modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -518,6 +533,16 @@ private void updateAssignmentInternal(ConfigurationNotificationEvent<byte[]> ass
                                 ),
                                 groupOptions
                         );
+
+                        replicaMgr.startReplica(grpId,
+                                new PartitionReplicaListener(
+                                        partitionStorage,
+                                        internalTbl.partitionRaftGroupService(partId),
```

Comment:
I'm not sure that you'll retrieve @NotNull here, because services are started later on.

Reply:
I moved `startReplica`.